Mathlib/Algebra/Module/Submodule/Map.lean

```diff
@@ -619,3 +619,25 @@ def compatibleMaps : Submodule R (N →ₗ[R] N₂) where
 #align submodule.compatible_maps Submodule.compatibleMaps
 
 end Submodule
+
+namespace LinearMap
+
+variable { M M' R : Type* } [Semiring R] [AddCommMonoid M] [AddCommMonoid M']
+  [Module R M] [Module R M']
+
+/-- A linear map between two modules restricts to a linear map from any submodule p of the
```

Comment:
Your function argument is called `S`:
```suggestion
/-- A linear map between two modules restricts to a linear map from any submodule `S` of the
```
(or change the function argument instead)